src/main/java/org/mifos/processor/bulk/schema/Transaction.java

```diff
@@ -1,9 +1,6 @@
 package org.mifos.processor.bulk.schema;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.*;
```

Comment:
@danishjamal104 Please use individual imports instead of '*'. Please fix this in other places as well.